flang/test/Semantics/cuf21.cuf

```diff
@@ -0,0 +1,28 @@
+! RUN: bbc -emit-hlfir -fcuda %s -o - | FileCheck %s
```

Comment:
We should use the semantic check run line here: 

```
! RUN: %python %S/test_errors.py %s %flang_fc1
```

We just want to see if semantic pass without error. We are not really interested in the IR.